ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/headervalidationrules/TimestampMoreRecentThanParent.java

```diff
@@ -45,7 +51,11 @@ private boolean validateTimestamp(final long timestamp, final long parentTimesta
   private boolean validateHeaderSufficientlyAheadOfParent(
       final long timestamp, final long parentTimestamp) {
     final long secondsSinceParent = timestamp - parentTimestamp;
-    if (secondsSinceParent < minimumSecondsSinceParent) {
+    if (usingExperimentalShortBFTBlockPeriod) {
```

Comment:
It would be better to disable this in the QbftBlockHeaderValidationRulesetFactory based on the config from QbftConfigOptions